test/deploy/linux/postgres/install/debian/roles/configure/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+---
+- debug:
+    msg: Install Postgres
+
+- name: Set default create_newrelic_user (default not create)
+  set_fact:
+    create_newrelic_user: "false"
+  when: create_newrelic_user is undefined
+
+- name: Set default create_env_var (default not create)
+  set_fact:
+    create_env_var: "false"
+  when: create_env_var is undefined
+
+- name: Install postgresql
+  apt:
+    name: ['postgresql']
+    update_cache: yes
+    state: latest
+  become: true
+
```

Comment:
The rhel PR is doing some additional work to [init the database and run postgres](https://github.com/newrelic/open-install-library/blob/postgresRhel/test/deploy/linux/postgres/install/rhel/roles/configure/tasks/main.yml#L29-L39). Just wondering, is that not necessary here?

Reply:
Not really no, it was necessary on rhel to make it work at all